packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -332,6 +354,20 @@ export default class extends Component {
                     {host.multipathing && <MultipathableSrs hostId={host.id} />}
                   </td>
                 </tr>
+                {this.state.schedGran !== 'MESSAGE_METHOD_UNKNOWN' && (
```

Comment:
You can test if the  `scheduleGranularity` isn't `undefined` instead of the message code.